app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -2,6 +2,7 @@ module Katello
   class Api::V2::ContentViewFilterRulesController < Api::V2::ApiController
     before_filter :find_filter
     before_filter :find_rule, :except => [:index, :create]
+    before_filter :create_rules, only: [:create]
```

Comment:
I'm okay with using ':' hash syntax in contained areas, but prefer to not mix it in the same block as the hash rocket :rocket: 
Since we're already somewhat mixed, consistency within a method or block of code should be our goal.